api/routes.js

```diff
@@ -308,4 +308,8 @@ exports.set = function(app, mongo, log) {
     app.get(URL_TEAM_PICK, (request, response) => {
         response.send(JSON.stringify(teamsJSON, null, 2));
     });
+
+    app.get('/api/healthcheck', (request, response) => {
```

Comment:
Try to not hardcode stuff in here, use the same terminology the other routes have.